src/sentry/api/serializers/models/group.py

```diff
@@ -909,6 +912,20 @@ def _get_seen_stats(self, item_list, user):
                 )
         return time_range_result
 
+    def _get_inbox_stats(self, item_list):
+        group_ids = [g.id for g in item_list]
+        group_inboxes = GroupInbox.objects.filter(group__in=group_ids)
+        inbox_stats = {
+            gi.group.id: {
```

Comment:
`gi.group_id` saves you fetching it from the group.

Reply:
Nice, thanks